docs/source/glossary.rst

```diff
@@ -7,25 +7,33 @@ Glossary
 .. glossary::
 
     Configuration
-        The configuration is a set of parameters that will define the level for each
-        noise type in a nested dictionary. The configuration is a hierarchical structure
-        and values are specific to each dataset, column, and noise type.
+        Settings that allow you to customize the noise present in the datasets
+        generated by pseudopeople.
+        Noise is configurable at a very fine-grained level, with settings
+        specific to the dataset, noise type, and column (where applicable).
 
     Datasets
-        Datasets contain un-noised data will be noised and returned to the
-        user with errors at a level provided by the configuration.
-        The noised data will be output in format specific to the dataset type,
-        eg a census or tax form.
+        The types of data that can be simulated with pseudopeople, each of which is
+        the simulated analog of a "real world" database of administrative records.
+        For example, pseudopeople's American Community Survey (ACS) dataset is analogous to the
+        data that would be collected by that survey in real life.
 
     Entity resolution (ER)
         The task of identifying the unique entities associated with a set of records,
         where multiple records may refer to the same entity.
         Also called "record linkage," among other names.
 
-    Noise functions
-        Functions that will be applied to datasets at a row or column level to apply
-        errors or mistakes in the data to replicate realistic challenges in personal
-        record linkage.
+    Noise
+        Errors introduced to a pseudopeople dataset.
+        These simulate data errors that would be found in real-life survey
+        and administrative data.
+
+    Noise types
+        The types of error that can be introduced to a pseudopeople dataset.
+        Each one simulates a specific type of mistake/inaccuracy that could occur in
```

Comment:
```suggestion
        Each one simulates a specific type of error that could occur in
```

Reply:
This feels a little repetitive with the previous sentence to me, is that intentional?

Reply:
I agree it sounds repetitive to say "error" again. How about instead of a slash you just say "mistake or inaccuracy"?

Also, is it true that noise is always "errors" or "mistakes" or "inaccuracies", or could it be some other kind of variation in the data? Nicknames seem like a borderline case to me in terms of whether the provided data is "wrong" or just an alternate version of the true data.

Reply:
Fair point, it is kind of borderline. But I can't think of a way to describe this that could include that nuance without becoming much more vague. We of course have more details elsewhere, but the glossary needs to be short and sweet, I think.

I'll replace the slash!

Reply:
Ah, I forgot that nicknames were my example 😅 I'll change it to "choose the wrong option"

Reply:
> Ah, I forgot that nicknames were my example 😅 I'll change it to "choose the wrong option"

I don't know if it's necessary to change the example -- maybe it's good to have some ambiguity...

On the Noise page I also describe noise as "errors", and I say it can also be called "corruption" or "distortion." Perhaps "distortion" has less connotation that something is "wrong." But our whole model of how pseudopeople works is based on the (false, or at least simplistic) idea that there is a single, unambiguous truth, and "noise" is something different from the "truth." So I don't know if there's an easy way to describe things more accurately without making everything a lot more complicated. And I think the noise functions we've implemented so far _do_ all fit into the category of "errors", even nicknames if you take the perspective that you're supposed to put your legal name on official documents, so using a nickname is a mistake.

Reply:
Totally agree!

I already changed the example, do you want to take a look at how it is now and let me know if you'd rather change it back? Then, I think I can merge this PR.